packages/teams-js/src/public/pages.ts

```diff
@@ -750,4 +750,94 @@ export namespace pages {
       return runtime.supports.pages ? (runtime.supports.pages.appButton ? true : false) : false;
     }
   }
+
+  /**
+   * Parameters for the NavigatewithinApp API
```

Comment:
I don't think I understand this comment, there's no API call called NavigateWithinApp. This interface should also be moved into the subcapability since that's where it's intended to be used.

Reply:
Resolved in latest commit